### DISCUSSION TITLE
Welcome to eFBL Discussions!

### DISCUSSION THREAD

**Discussion by LuceliaTinembart** [FIATA/eFBL#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sherifanwar80** [comment]
Hello,
I am Sherif from InfoSystech for software technologies. We want to implement your eFBL. I need to join your communtiy and share in your discussions.

Thanks,
Sherif
